lib/tasks/import_remediations.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+desc 'Update rule remediation availability by querying the Remdiations API'
```

Comment:
IMO it'd also be good to show the usage with the env variable here in the description.